src/status_im/ui/components/status_bar/styles.cljs

```diff
@@ -82,7 +82,7 @@
 
 (defstyle view-wallet
   {:ios     (create-view-style {:background-color styles/color-blue4})
-   :android (create-view-style {:background-color styles/color-blue5})})
+   :android (create-view-style {:background-color styles/color-blue4})})
```

Comment:
`defstyle` isn't needed here , no difference in styles

Reply:
:+1: